### PR TITLE
ImageMagick: update to 6.9.12-98

### PR DESCRIPTION
**Pull request by mohd-akram**
#### Description

Dependents bumped via `port echo depends_lib:ImageMagick | xargs ./port-revbump`. Script [here](https://gist.github.com/mohd-akram/92912470caec3c6bc99dea748eab3779).

###### Tested on
macOS 13.6 22G120 x86_64
Xcode 15.0 15A240d

###### Verification
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by mascguy** [comment]
Mohamed, you can drop the rev-bump for `inkscape-devel`, as it's currently broken. (I'm aware of it, but haven't had a chance to fix yet.)

File: perl/p5-perlmagick/Portfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ PortGroup           perl5 1.0
 epoch               1
 perl5.branches      5.28 5.30 5.32 5.34
 perl5.setup         PerlMagick 6.9.11-60
-revision            0
+revision            1
```
Comment (ryandesign) [inline perl/p5-perlmagick/Portfile:11]: The version of p5-perlmagick should be updated to match ImageMagick rather than increasing its revision.

**Comment by jmroot** [comment]
Please rebase against master to pick up a789b83f

File: perl/p5-perlmagick/Portfile (hunk 1 above)
Comment (mohd-akram) [reply perl/p5-perlmagick/Portfile]: Thanks, fixed.

**Comment by mohd-akram** [comment]
Rebased to fix conflict.

**Comment by neverpanic** [comment]
>     * dblatex (@neverpanic) - destroot failing (python setup.py issue)

That's unrelated, feel free to skip the revbump for dblatex. I'll take a look at the problem separately.

**Pull request closed without merging**